test/functional/JIT_Test/src/jit/test/tr/SeqLoadSimplification/SeqLoadSimplificationTest.java

```diff
@@ -0,0 +1,935 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package jit.test.tr.SeqLoadSimplification;
```

Comment:
Does this need to be in it's own directory? Do we envision adding more files here? If not we should move it out.

Great set of tests by the way! Readers take note.

Reply:
Thanks.

I don't expect to add more files here. However, I don't really see a better place to put it. I tried looking at other tests to see how they organize their files but a lot of them are single files in a directory as well.

Reply:
Can we simply move `SeqLoadSimplificationTest.java` into `test/functional/JIT_Test/src/jit/test/tr/`?

Reply:
Yes, that's possible. In my latest changed I've now moved it out.